docs/design/values.md

```diff
@@ -232,28 +259,21 @@ Carbon:
 -   Dereferenced [pointers](#pointers): `*p`
 -   Names of subobjects through member access to some other durable reference
     expression: `x.member` or `p->member`
--   [Indexing](/docs/design/expressions/indexing.md): `array[i]`
+-   [Indexing](/docs/design/expressions/indexing.md) some other durable
+    reference expression: `array[i]`
```

Comment:
This is a subset of the criteria from `indexing.md`. In particular, indexing into value expression with a type like C++'s `std::span` should also produce a durable reference expression